contributing/TESTING.md

```diff
@@ -43,99 +43,97 @@ Before running these commands, make sure you have Node.js and Gulp installed. Fo
-| `gulp check-types`                                        | Verifies that there are no errors associated with Closure typing. Run automatically upon push.                                                                                                                                                           |
-| `gulp dep-check`                                          | Runs a dependency check on each module. Run automatically upon push.                                                                                                                                                                                     |
-| `gulp presubmit`                                          | Run validation against files to check for forbidden and required terms. Run automatically upon push.                                                                                                                                                     |
-| `gulp validator`                                          | Builds and tests the AMP validator. Run automatically upon push.                                                                                                                                                                                         |
-| `gulp ava`                                                | Run node tests for tasks and offline/node code using [ava](https://github.com/avajs/ava).                                                                                                                                                                |
-| `gulp todos:find-closed`                                  | Find `TODO`s in code for issues that have been closed.                                                                                                                                                                                                   |
-| `gulp visual-diff`                                        | Runs all visual diff tests on a headless instance of local Chrome after building the unminified runtime with the `prod` version of `AMP_CONFIG`. Requires `PERCY_TOKEN` to be set as an environment variable or passed to the task with `--percy_token`. |
-| `gulp visual-diff --compiled`                             | Same as above, but builds the minified runtime.                                                                                                                                                                                                          |
-| `gulp visual-diff --config=<config>`                      | Same as above, but `config` can be `prod` or `canary`. (Defaults to `prod`.)                                                                                                                                                                             |
```

Comment:
Pretty sure the `--config` option still applies here, since it will internally do a `dist` with  the config that was passed in.

Reply:
Reverted this line